test/testmachinery/system/shoot_cp_migration/migrate_test.go

```diff
@@ -113,6 +115,10 @@ func beforeMigration(ctx context.Context, t *ShootMigrationTest, guestBookApp *a
 		return nil
 	}
 
+	if !v1beta1helper.NginxIngressEnabled(t.Shoot.Spec.Addons) {
+		return fmt.Errorf("the shoot must have the ngninx-ingress addon enabled")
+	}
```

Comment:
We should now be able to drop https://github.com/gardener/gardener/blob/8d66afe33a1792e1c9b93defbfe35cbedc4007da/test/testmachinery/system/shoot_cp_migration/migrate_test.go#L190-L209